docs/pages/blog/2023-discord-announcement.md

```diff
@@ -0,0 +1,55 @@
+---
+title: 'MUI is now on Discord!'
+description: Come join our community to engage in lively discussions, share your projects, and interact with the MUI team
+date: 2023-08-02T00:00:00.000Z
+authors: ['richbustos']
+tags: ['News']
+card: true
+---
+
+<a href="https://mui.com/r/discord"><img src="/static/blog/2023-discord-announcement/discord.png" width="1280" height="640" style="margin-bottom: 16px;" alt="Discord banner with link" /></a>
+
+Creating a place where we can interact with our community has been our priority for this year.
+Majority of our discussions and interactions with you have been through our [Twitter](https://twitter.com/MUI_hq), [GitHub](https://github.com/mui), and [marketing page](https://mui.com/).
```

Comment:
Let's underscore the "scattered" nature of our community pre-Discord; drop "marketing page" (that's an internal term that won't resonate with readers) and add LinkedIn and SO.

```suggestion
Up until now, our community has been scattered across discussions and interactions on [Twitter](https://twitter.com/MUI_hq), [LinkedIn](https://linkedin.com/company/mui), [GitHub](https://github.com/mui), and [Stack Overflow](https://stackoverflow.com/questions/tagged/material-ui).
```